smartsim/_core/control/controller.py

```diff
@@ -586,3 +594,38 @@ def reload_saved_db(self, checkpoint_file):
         finally:
             JM_LOCK.release()
 
+
+    def _set_dbobjects(self, manifest):
+        db_addresses = self._jobs.get_db_host_addresses()
+        cluster = len(db_addresses) > 1
+        address = db_addresses[0]
+
+        try:
+            client = Client(address=address, cluster=cluster)
```

Comment:
So this should only be reachable in the case where we think the orchestrator is active but it's actually not active correct? 

could we re-use ``check_cluster_status``? or split off the orchestrator method for checking existance? lets chat about this one today.

Reply:
Yep, right